content/technotes/2019-01-10-hugo.md

```diff
@@ -0,0 +1,101 @@
+
+And should you ever get lost, we've now a [cool 404 page](/404), although mentioning it probably lessens its coolness.
+
+# More content
+
+In the previous section we mention the [community calls living in ropensci.org](/commcalls). They used to have their own source and subdomain, but we've transferred them which has several advantages:
+
+* When reading about community calls, you're looking at the same design than when reading tech notes which makes us happy since we like our website. Besides, thanks to the footer and header, you're more prone to bouncing on over parts of the website. The [community calls list](/commcalls) list and [individual pages](https://ropensci.org/commcalls/2018-12-18/) use partials from the rest of the website, and our Hugo experience. In each comm call page, the video recording is for instance embedded!
+
+* Community calls now being similar to blog posts and tech notes, they're cross-listed via their tags and authors so are more likely to be found.
+
+# Feed updates
+
+If you want to keep up with our production, you can subscribe to different feeds (or [our newsletter](https://news.ropensci.org/) since it includes mentions of recent posts!).
+
+## All the feeds
+
+Our website has both RSS and JSON feeds. We added a JSON feed because we could and because some people might like that. 
+
+* RSS feeds https://ropensci.org/blog/index.xml, https://ropensci.org/technotes/index.xml
+
+* JSON feeds https://ropensci.org/blog/index.json, https://ropensci.org/technotes/index.json
+
+Here's [how we added the JSON feeds](https://github.com/ropensci/roweb2/pull/315).
+
+## Not too much content for R-Bloggers
+
+R-Bloggers feed https://ropensci.org/rbloggers/index.xml
+
+We created an [R-Bloggers page which only outputs XML](https://github.com/ropensci/roweb2/blob/master/content/rbloggers/_index.md), with [an XML template](https://github.com/ropensci/roweb2/blob/master/themes/ropensci/layouts/rbloggers/rss.xml) ensuring this RSS feed is a full RSS feed contrary to the rest of our website that uses Hugo default RSS feeds with excerpts. We do not filter posts by category which means this not-so-R post (but still a bit R, thanks to our explaining how to browse our R stuff ;o) will appear in R-Bloggers, but we might do that later on.
+
+# Twitter friendliness
+
+We're aware that our content does not only get shared on Twitter, but this social media platform is an important player so we've upped our Twitter game with two edits.
```

Comment:
"We're aware that our content does not only get shared on Twitter"  is hard to parse "while twitter is not the only platform on which our content is shared, ..."